Rails.gitignore

```diff
@@ -14,7 +14,7 @@ pickle-email-*.html
 
 # TODO Comment out these rules if you are OK with secrets being uploaded to the repo
 config/initializers/secret_token.rb
-config/secrets.yml
+# config/secrets.yml # only include if you have production secrets in this file, which is no longer a Rails default
```

Comment:
Move this comment up to the line above so it's consistent with others.

Feel free to add a newline as well if you'd like to separate it from the previous rule.